src/components/data-management/ProjectsList.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Space } from 'antd';
+
+import ProjectDeleteModal from './ProjectDeleteModal';
+import { setActiveProject, updateProject, deleteProject as deleteProjectAction } from '../../redux/actions/projects';
+import ProjectCard from './ProjectCard';
+import validateInputs, { rules } from '../../utils/validateInputs';
+
+const validationChecks = [
+  rules.MIN_8_CHARS,
+  rules.MIN_2_SEQUENTIAL_CHARS,
+  rules.ALPHANUM_DASH_SPACE,
+  rules.UNIQUE_NAME_CASE_INSENSITIVE,
+];
+
+const ProjectsListContainer = (props) => {
+  const { height, filter } = props;
```

Comment:
Is there some way we can find out the `height` without receiving it from outside?

Reply:
I had a quick search on this matter. It seems that we may be able to use a `ref` that points to an HTML element that's rendered, and get the `ref.current.offsetWidth` value to get the width. I tried this on the elements rendered by this function, but it returns undefined. According to [this comment](https://stackoverflow.com/a/64814269/1940886) on StackOverflow, using the above method we may be able to get the initial width of the object. However, to listen we still need to set up a resize listener. We currently do this using the `ReactResizeObserver` HOC component, which is the source of the `height` props seen here.